src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -329,15 +329,25 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
     */
     @AuraEnabled
     public static String saveAndDryRunRow(Id batchId, DataImport__c dataImport) {
+
         try {
+
             if (String.isNotBlank(dataImport.Id)) {
                 Database.update(dataImport);
             } else {
                 Database.insert(dataImport);
             }
+
             return runSingleDryRun(dataImport.Id, batchId);
+
         } catch (Exception ex) {
-            throw new AuraHandledException(ex.getMessage());
+
+            // create custom lwc exception and throw it
```

Comment:
@kenlewis-sfdo - if you get a chance, maybe just be a second set of eyes here to make sure we aren't causing negative consequence to our legacy BGE product with this update?

Reply:
This was needed if we wanted to manage the exception in the same way as SGE, excellent catch Patrick. Hopefully didn't broke existent functionality

Reply:
This should be ok because this is a new method that is only used for the new GE.  This method should probably be located in `GE_BatchGiftEntryController`.  If you guys agree and it isn't too much work maybe we could move it there as part of this PR.

Reply:
I think that makes sense to move this method to GE_BatchGiftEntryController - that's been throwing me because I assume everything in this class is legacy! @grivero do you think you can move this, and then we should be good to go!

Reply:
Thanks for reviewing @kenlewis-sfdo. 
Regarding moving method: If that's the way to go I'll move that and any related Unit Tests to `GE_BatchGiftEntryController` right after our Sprint Review meeting